src/utils/hooks/useRouter.tsx

```diff
@@ -104,3 +106,8 @@ export function useRouter(): RouterResult {
     }
   }, [location])
 }
+
+const parseQueryString = (queryString: string): { [key: string]: string } => {
+  const queries = queryString.split('&').map((query) => query.split('='))
+  return Object.fromEntries(queries)
+}
```

Comment:
Small enhancement to the "updateQuery" functionality to accept a query string directly (e.g. `isActive=false`) as well as an object of key/values (e.g. `{ isActive: false }` )